modules/flowable-cmmn-json-converter/src/main/java/org/flowable/cmmn/editor/json/converter/ServiceTaskJsonConverter.java

```diff
@@ -142,6 +142,10 @@ protected BaseElement convertJsonToElement(JsonNode elementNode, JsonNode modelN
             task.setResultVariableName(CmmnJsonConverterUtil.getPropertyValueAsString(PROPERTY_SERVICETASK_RESULT_VARIABLE, elementNode));
         }
 
+        if (CmmnJsonConverterUtil.getPropertyValueAsBoolean(PROPERTY_SERVICETASK_STORE_RESULT_AS_TRANSIENT, elementNode)) {
```

Comment:
Isn't the reverse missing? When importing from the BPMN XML into the Modeler JSON?

Reply:
You're right, that slipped through the cracks. I added it.